daemon/rpc/handlers.go

```diff
@@ -327,6 +331,7 @@ func (s *Server) GetSwapOut(ctx context.Context, req *GetSwapOutRequest) (*GetSw
 		Invoice:            swap.Invoice,
 		InputAmount:        swap.InputAmount.InexactFloat64(),
 		OutputAmount:       swap.OutputAmount.InexactFloat64(),
+		Outcome:            (*string)(&swap.Outcome),
```

Comment:
line 301 does it like this, is it possible?
```suggestion
		Outcome:            &swap.Outcome,
```

Reply:
Sorry, I thought I answered this :( this one is returning the enum and the other one a string, will do both enums